### ISSUE TITLE
tree doesn't expand all subdirectories

### ISSUE THREAD

**Issue by pbedia-stratio** [mlorenzo-stratio/vcli#2]
```
[200] vcli@vault.service.paas.labs.stratio.com:8200 > tree /
/
+ ca-trust/
+ cubbyhole/
+ dcs/
+ gosec-sso/
+ gosec/
+ people/
+ secret/
+ sys/
+ userland/

```

**Comment by ml0renz0** [comment]
It seems to [work perfectly fine for me](https://asciinema.org/a/70qWOTEknavZmlHONHb3iYOA2)
Are you running this inside a docker container?

**Comment by pbedia-stratio** [comment]
Yes, i'm running it inside a docker container (eos-installer one).

**Comment by ml0renz0** [comment]
Fixed on 5d04926
